tests/unit_tests/external_interaction/callbacks/conftest.py

```diff
@@ -159,6 +159,8 @@ class TestData:
             "oav_snapshot_top_left_y": 100,
             "oav_snapshot_num_boxes_x": 40,
             "oav_snapshot_num_boxes_y": 20,
+            "oav_snapshot_microns_per_pixel_x": 1.25,
+            "oav_snapshot_microns_per_pixel_y": 1.25,
```

Comment:
Could: Would be good to not have these equal in each dimension, to test that we're using the right one at the right time